spartan/scripts/deploy_network.sh

```
@@ -288,17 +290,18 @@ PROVER_REAL_PROOFS = ${PROVER_REAL_PROOFS}
 TRANSACTIONS_DISABLED = ${TRANSACTIONS_DISABLED:-null}
 NETWORK = "${NETWORK:-null}"
 STORE_SNAPSHOT_URL = "${STORE_SNAPSHOT_URL}"
-
-# Bot settings
 BOT_RESOURCE_PROFILE = "${BOT_RESOURCE_PROFILE}"
+BOT_MNEMONIC = "${LABS_INFRA_MNEMONIC}"
+BOT_TRANSFERS_MNEMONIC_START_INDEX = ${PROVER_MNEMONIC_START_INDEX}
 BOT_TRANSFERS_REPLICAS = ${BOT_TRANSFERS_REPLICAS}
 BOT_TRANSFERS_TX_INTERVAL_SECONDS = ${BOT_TRANSFERS_TX_INTERVAL_SECONDS}
 BOT_TRANSFERS_FOLLOW_CHAIN = "${BOT_TRANSFERS_FOLLOW_CHAIN}"
+BOT_SWAPS_MNEMONIC_START_INDEX = ${PROVER_MNEMONIC_START_INDEX}
 BOT_SWAPS_REPLICAS = ${BOT_SWAPS_REPLICAS}
 BOT_SWAPS_TX_INTERVAL_SECONDS = ${BOT_SWAPS_TX_INTERVAL_SECONDS}
 BOT_SWAPS_FOLLOW_CHAIN = "${BOT_SWAPS_FOLLOW_CHAIN}"
-BOT_TRANSFERS_PRIVATE_KEY = "${BOT_TRANSFERS_PRIVATE_KEY:-0xcafe01}"
-BOT_SWAPS_PRIVATE_KEY = "${BOT_SWAPS_PRIVATE_KEY:-0xcafe02}"
+BOT_TRANSFERS_L2_PRIVATE_KEY = "${BOT_TRANSFERS_L2_PRIVATE_KEY:-0xcafe01}"
+BOT_SWAPS_L2_PRIVATE_KEY = "${BOT_SWAPS_L2_PRIVATE_KEY:-0xcafe02}"
```

I see this was already here, but what's the purpose of defaulting to invalid private key?

---

Cleared up over slack, the key looks invalid but it's parsed as a Fr so it adds leading 0s as necessary.